README.md

```diff
@@ -8,8 +10,75 @@
 
 ***Due to limitations imposed by Github and occasional slow server response times, some badges may require a page refresh to load.**
```

Comment:
Noticing that there's unequal emphasis here (*** ... **) which causes GitHub preview to think that the rest of the README text should be bolded

Reply:
Good catch. Got it.